### PR TITLE
DRYs up the find this item partial.

### PR DESCRIPTION
**Pull request by bwatson78**
1. Uses the value pulling logic in a new method defined to just handle the persisted URL on this page.

### CODE REVIEW

File: app/views/catalog/_find_this_item_block.html.erb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <% doc_presenter = show_presenter(document) %>
 <% fields = presenter.new(document: doc_presenter.fields_to_render).terms %>
+<% def purl doc_id
```
Comment (wulman16) [inline app/views/catalog/_find_this_item_block.html.erb:3]: Is it possible to define this new method in a helper file instead of a view file?

Comment (wulman16) [reply app/views/catalog/_find_this_item_block.html.erb:3]: Or maybe in the presenter

Comment (bwatson78) [reply app/views/catalog/_find_this_item_block.html.erb:3]: Will do.

**Comment by bwatson78** [comment]
@wulman16 Update pushed.

**Pull request merged**